src/components/dialogs/inputs/input-hooks.js

```diff
@@ -864,3 +736,81 @@ export const useCSVReader = ({ label, header }) => {
     }, [selectedFile, intl, header]);
     return [selectedFile, setSelectedFile, field, fileError];
 };
+
+export const useRadioValue = ({
+    label,
+    possibleValues = [],
+    defaultValue,
+    id,
+    validation = {},
+    inputForm,
+    doTranslation = true,
+}) => {
+    const [value, setValue] = useState(defaultValue);
+    const intl = useIntl();
+
+    useEffect(() => {
+        // Updates the component when the correct default value is given by the parent component.
+        if (defaultValue !== undefined && defaultValue.length > 0) {
```

Comment:
Can be simplified to : `defaultValue?.length > 0`

Reply:
Yes, thank you.